app/views/bookings/index.html.erb

```diff
@@ -5,18 +5,12 @@
     <% @bookings.each do |booking| %>
       <% if booking.user == current_user %>
        <%# this if checks that the user is the person booking the dress %>
-      <% booked_dress = Dress.find(booking.dress_id) %>
+      <% booked_dress = booking.dress %>
```

Comment:
Why are you setting this variable here? Can we just directly use booking.dress instead?